impl/maven-impl/src/main/java/org/apache/maven/impl/model/DefaultModelInterpolator.java

```diff
@@ -109,8 +109,7 @@ private InnerInterpolator createInterpolator(
             Model model, Path projectDir, ModelBuilderRequest request, ModelProblemCollector problems) {
 
         Map<String, Optional<String>> cache = new HashMap<>();
-        Function<String, Optional<String>> ucb =
-                v -> Optional.ofNullable(callback(model, projectDir, request, problems, v));
+        Function<String, Optional<String>> ucb = v -> Optional.ofNullable(doCallback(model, projectDir, request, v));
```

Comment:
```suggestion
        Function<String, Optional<String>> ucb = v -> Optional.ofNullable(callback(model, projectDir, request, v));
```
cleanup in dedicated pr:
- https://github.com/apache/maven/pull/2374

Reply:
<img width="971" alt="image" src="https://github.com/user-attachments/assets/c857b7c4-da92-430c-99f1-9f1ef769cb16" />